CHANGES.rst

```diff
@@ -342,6 +345,10 @@ API changes
 
 - ``astropy.coordinates``
 
+  - ``SphericalRepresentation`` now has a ``._unit_representation`` class attribute to specify 
```

Comment:
I don't think you need this section here.  It's not obvious that this is what it means, but ``API Changes`` here are generally changes to *existing* pieces of the API.  This is instead an *addition*, so it only need go in "new features".